fbu-app/Cells/ReminderCell.h

```diff
@@ -13,6 +13,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ReminderCell : UITableViewCell
 
+@property (weak, nonatomic) IBOutlet UIButton *checkmarkButton;
```

Comment:
variables don't have to be made properties, you can make them instance variables instead